src/Mvc/Mvc.RazorPages/src/Builder/RazorPagesEndpointRouteBuilderExtensions.cs

```diff
@@ -337,18 +350,51 @@ public static void MapDynamicPageRoute<TTransformer>(this IEndpointRouteBuilder
             EnsureRazorPagesServices(endpoints);
 
             // Called for side-effect to make sure that the data source is registered.
-            GetOrCreateDataSource(endpoints).CreateInertEndpoints = true;
```

Comment:
Is it fine that `CreateInertEndpoints` is no longer being set?

Reply:
It's set inside the method below